### PR TITLE
Update sample README to remove unnecessary contentType parameters

### LINKED ISSUE #393: The Batch-events sample does not require contentType
As of SCSt 2.0, content type is now always application/json.    Please remove content type from README.

### PR DESCRIPTION
**Pull request by royclarkson**
Resolves #393

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@royclarkson Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-cloud/spring-cloud-task&pullRequestId=394)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=spring-cloud/spring-cloud-task&pullRequestId=394) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

File: spring-cloud-task-samples/batch-events/README.adoc [hunk 1]
```diff
@@ -27,11 +27,10 @@ $ ./mvnw clean install
 
 [source,shell,indent=2]
 ----
-$ java -jar target/batch-events-1.2.1.RELEASE.jar --spring.cloud.stream.bindings.batch-events.contentType=application/json
+$ java -jar target/batch-events-1.2.1.RELEASE.jar
 ----
 
-For example you can listen for specific job execution events on a specified channel with a Spring Cloud Stream Sink
-like the log sink using the following:
+For example you can listen for specific job execution events on a specified channel with a Spring Cloud Stream Sink like the log sink using the following:
```
Comment (cppwfs) [inline spring-cloud-task-samples/batch-events/README.adoc]: Please readd C/R.

File: spring-cloud-task-samples/task-events/README.adoc [hunk 2]
```diff
@@ -17,11 +17,10 @@ $ ./mvnw clean install
 
 [source,shell,indent=2]
 ----
-$ java -jar target/task-events-1.2.1.RELEASE.jar --spring.cloud.stream.bindings.task-events.contentType=application/json
+$ java -jar target/task-events-1.2.1.RELEASE.jar
 ----
 
-You can listen for the events on the task-events channel with a Spring Cloud Stream Sink
-like the log sink using the following:
+You can listen for the events on the task-events channel with a Spring Cloud Stream Sink like the log sink using the following:
```
Comment (cppwfs) [inline spring-cloud-task-samples/task-events/README.adoc]: Please add C/R.

**Comment by cppwfs** [comment]
LGTM Thanks for contribution.

**Pull request closed without merging**